storing-app-data/slides.md

```diff
@@ -0,0 +1,456 @@
+theme: Plain Jane, 3
+
+# Storing app data reliably, performantly and easily
+
+<br>
+
+Kurt Nelson
+Mobile Platform at Uber
+
+[.header: alignment(left), text-scale(1.5)]
+[.autoscale: true]
+
+^ Hi! I'm Kurt and I work at Uber on the mobile platform team. Our job is to make mobile developer's jobs easier while improving the quality of our apps. I used to do a very similar job at Google, and I'm here to talk about something I have worked on at both companies.
+
+---
+
+# Apps written at Uber & Google are deployed at scale.
+
+^ Both Uber & Google have multiple apps used by millions (or billions) of users daily. We use the term "at scale" when talking about the parts of our development process that ensure we maintain our user experience.
```

Comment:
Fairly specific and implied exclusive wording "Uber & Google", how about "Apps written at companies like Uber, Google, and Facebook are deployed at scale"